cartesian_controller_utilities/package.xml

```diff
@@ -12,20 +12,19 @@
 
   <!-- One license tag required, multiple allowed, one license per tag -->
   <!-- Commonly used license strings: -->
-  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
-  <license>TODO</license>
+  <license>BSD</license>
```

Comment:
Which BSD variant is this? If 3-clause, might be nice to use the SPDX identifier: `BSD-3-Clause`.

Reply:
@gavanderhoorn Good point. Yes, it's BSD-3-clause. After checking this repository, it's probably best to update this for all packages on both `master` and `ros2`.